lib/active_merchant/billing/gateways/ixopay.rb

```diff
@@ -1,53 +1,40 @@
 module ActiveMerchant #:nodoc:
   module Billing #:nodoc:
     class IxopayGateway < Gateway
```

Comment:
(Discussed offline) The diff is against the raw output of the gateway generator. That code forms the baseline of our ```ixopay_integration``` branch; there is no PR for it since we don't want to merge it until Ixopay is in an appropriate state to be added to Active Merchant. The baseline code can be viewed here: https://github.com/activemerchant/active_merchant/compare/ixopay_integration#diff-51a89ecca18e8620ad6cb8e059787833